conan/cli/commands/graph.py

```diff
@@ -298,3 +300,110 @@ def graph_explain(conan_api, parser,  subparser, *args):
+                           help='Whether the provided reference is a build-require')
+    args = parser.parse_args(*args)
+    # parameter validation
+    validate_common_graph_args(args)
+    cwd = os.getcwd()
+    path = conan_api.local.get_conanfile_path(args.path, cwd, py=None) if args.path else None
+
+    # Basic collaborators, remotes, lockfile, profiles
+    remotes = conan_api.remotes.list(args.remote) if not args.no_remote else []
+    overrides = eval(args.lockfile_overrides) if args.lockfile_overrides else None
+    lockfile = conan_api.lockfile.get_lockfile(lockfile=args.lockfile,
+                                               conanfile_path=path,
+                                               cwd=cwd,
+                                               partial=args.lockfile_partial,
+                                               overrides=overrides)
+    profile_host, profile_build = conan_api.profiles.get_profiles_from_args(args)
+
+    if path:
+        deps_graph = conan_api.graph.load_graph_consumer(path, args.name, args.version,
+                                                         args.user, args.channel,
+                                                         profile_host, profile_build, lockfile,
+                                                         remotes, args.update,
+                                                         check_updates=args.check_updates,
+                                                         is_build_require=args.build_require)
+    else:
+        deps_graph = conan_api.graph.load_graph_requires(args.requires, args.tool_requires,
+                                                         profile_host, profile_build, lockfile,
+                                                         remotes, args.update,
+                                                         check_updates=args.check_updates)
+    print_graph_basic(deps_graph)
+
+    dependencies = deps_graph.nodes[1:]
+    resolved_ranges = list(deps_graph.resolved_ranges.keys())
+
+    latest_recipe = []
+
+    ConanOutput().title("Checking remotes")
+
+    # Data structure to solve the repeated dependencies in graph
+    dict_nodes = {}
+    for node in dependencies:
+        if node.name in dict_nodes:
+            dict_nodes[node.name].append(node)
+        else:
+            dict_nodes[node.name] = [node]
+
+    for node_name, node_list in dict_nodes.items():
+        ref_pattern = ListPattern(node_name, rrev=None, prev=None)
+        latest_remote_ref, found_remote, recipe_ref = None, None, None
+        for remote in remotes:
+            try:
+                remote_ref_list = conan_api.list.select(ref_pattern, package_query=None,
+                                                        remote=remote)
+                if not remote_ref_list.recipes:
+                    continue
+            except Exception:
+                ConanOutput(f"{node_name} not found in remote {remote.name}")
+                continue
```

Comment:
Too broad exception. If the repo is off-line this should actually raise an error an stop.
Better:
- Only capture specific exceptions
- The ``ConanOutput(msg)`` is wrong, this means this is at least not tested.